.codesandbox/sandbox/src/end-to-end-test-helpers.ts

```diff
@@ -1,13 +1,13 @@
 import { Session } from "@inrupt/solid-client-authn-browser";
 import { WebsocketNotification } from "@inrupt/solid-client-notifications";
 
-const gateway = "https://notification.pod.inrupt.com/";
+const DEFAULT_GATEWAY = "https://notification.pod.inrupt.com/";
 
 export function getHelpers(podRoot: string, session: Session) {
-  async function connectWebsocket() {
+  async function connectWebsocket(gateway: string = DEFAULT_GATEWAY) {
     const notification = new WebsocketNotification(podRoot, {
       fetch: session.fetch,
-      gateway,
+      gateway: DEFAULT_GATEWAY,
```

Comment:
```suggestion
      gateway,
```
I'm a little confused, isn't the DEFAULT_GATEWAY already set in the function parameter here?

Reply:
Absolutely, that's a typo: I didn't intend to change the parameter, but it got updated when I used the IDE to change what initially was called `gateway` into `DEFAULT_GATEWAY` ^^

Reply:
12dbd8e